service-01/ErrorCodes.acn

```diff
@@ -23,10 +23,37 @@
 
 ErrorCodes DEFINITIONS ::= BEGIN
 
--- FailureNotice types
-FailureCode [] --TODO
+AcceptanceFailureCode [] --TODO
```

Comment:
Fixed as suggested, but maybe it's better idea to separate each FailureNotice and ErrorCodes than keep all types in one module? Will be much more readable and easier for user to specify each ErrorCodes